test/bin/test-citgm.js

```diff
@@ -79,8 +82,12 @@ test('bin: sigterm', (t) => {
   proc.stdout.once('data', () => {
     proc.kill('SIGINT');
   });
-  proc.on('exit', (code) => {
-    t.equal(code, 1, 'omg-i-pass should fail from a sigint');
+  proc.on('exit', (code, signal) => {
+    if (isWin32) {
+      t.equal(signal, 'SIGINT', 'omg-i-pass should fail from a sigint');
```

Comment:
Is `signal` not `SIGINT` on the other platforms?

Reply:
No, on unixes `signal` is `null`. That's because `citgm` has a `SIGINT` handler that cleans up the temporary directory and exit.